libs/platform/src/lib/table/table.component.ts

```diff
@@ -849,6 +853,10 @@ export class TableComponent<T = any> extends Table<T> implements AfterViewInit,
         this._listenToTableContainerMouseLeave();
 
         this._cdr.detectChanges();
+
+        if (this.expandOnInit === true) {
```

Comment:
```suggestion
        if (this.expandOnInit) {
```
Why bother with strict `true` comparison though